lib/rdoc/markup.rb

```diff
@@ -99,355 +99,6 @@
 #
 # See RDoc::MarkupReference.
 #
-# === Escaping Text Markup
```

Comment:
I don't see this information in `RDoc::MarkupReference`. This section should be incorporated in the `RDoc::MarkupReference` file so we don't lose information.

Reply:
Thanks, @peterzhu2118, for catching this important error.  In studying the omitted text, I've had some interesting insights about the rest of this page, which I'm still thinking about.  For the moment, I've just ported the omitted text without change.  More to come.